idaes/core/ui/fsvis/static/js/main.js

```diff
@@ -135,7 +143,66 @@ export class App {
     }
 
     /**
-     * Save the model value.
+     * Get the save time interval value from the application's setting block.
+     */
+    getSaveTimeInterval() {
+        let settings_url = "/setting?setting_key=".concat(this._save_time_interval_key);
+
+        let save_time_interval = this._default_save_time_interval;
+
+        $.ajax({url: settings_url, type: 'GET', contentType: "application/json"})
+            // On failure inform user and stop
+            .fail(error => this.informUser(
+                2, "Fatal error: cannot get setting value: " + error))
```

Comment:
not sure this should stop the application, since you can have a default value and survive.